### PR TITLE
Proofpoint TAP - fix URL parsing

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**
https://github.com/SekoiaLab/integration/issues/185

### CODE REVIEW

File: ProofPoint/proofpoint-tap/ingest/parser.yml [hunk 1]
```diff
@@ -94,7 +101,7 @@ stages:
                   {% elif threat.get("threatType", "").lower() == "url" %}
                   "type": "domain-name",
                   "url": {
-                    "domain": "{{threat.threat}}"
+                    "original": "{{threat.threat}}"
```
Comment (squioc) [inline ProofPoint/proofpoint-tap/ingest/parser.yml:104]: Can you add the `url.domain` with the same value in order to do a smooth transition? The field `threat.enrichments.indicator.url.domain` is currently used.

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the fix.
However, can you keep `threat.enrichments.indicator.url.domain` in addition to `threat.enrichments.indicator.url.original` and create another PR that remove `threat.enrichments.indicator.url.domain`?

**Comment by lvoloshyn-sekoia** [comment]
PR with remove - https://github.com/SEKOIA-IO/intake-formats/pull/1261

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**